api/lib/services/csvwBuilder.spec.ts

```diff
@@ -176,7 +176,7 @@ describe('csvwBuilder', () => {
       const datatypeMappings: Array<[NamedNode, string]> = [
         [xsd.double, 'number'],
         [xsd.base64Binary, 'binary'],
-        [xsd.dateTime, 'datetime'],
+        [xsd.dateTime, 'dateTime'],
```

Comment:
I don't think we want to drop support for "datetime" in the API.

Reply:
I'm probably missing something here and I'm trying to learn something. Isn't this testing that `xsd.dateTime` gets converted to CSVW `datetime` (the alias to CSVW `dateTime`)?

What was the reason to prefer the alias instead of the 'base' term borrowed from xsd?

Reply:
Not exactly sure, but I wouldn't change the CSVW generator logic just to change what users see in the UI

Reply:
Right, it's unrelated, I'll revert these two. I'm still curious about the reason if there's any, ping @tpluscode if there's something I can learn from this!